Run `make generated` to sync packages

zhengxiexie opened the pull request:
Sync pkg/apis/nsx.vmware.com/v1alpha1/2 with pkg/apis/v1alpha1/2.

timdengyun commented:
why I run ake generated, pkg/client/go.mod and pkg/client/go.sum also deleted? is there some change? 
	modified:   pkg/apis/nsx.vmware.com/v1alpha1/nsxserviceaccount_types.go
	modified:   pkg/apis/nsx.vmware.com/v1alpha1/vpc_types.go
	modified:   pkg/apis/nsx.vmware.com/v1alpha1/zz_generated.deepcopy.go
	modified:   pkg/apis/nsx.vmware.com/v1alpha2/ippool_types.go
	deleted:    pkg/client/go.mod
	deleted:    pkg/client/go.sum

zhengxiexie commented:
> why I run ake generated, pkg/client/go.mod and pkg/client/go.sum also deleted? is there some change? modified: pkg/apis/nsx.vmware.com/v1alpha1/nsxserviceaccount_types.go modified: pkg/apis/nsx.vmware.com/v1alpha1/vpc_types.go modified: pkg/apis/nsx.vmware.com/v1alpha1/zz_generated.deepcopy.go modified: pkg/apis/nsx.vmware.com/v1alpha2/ippool_types.go deleted: pkg/client/go.mod deleted: pkg/client/go.sum

Yes, me too, fixed it, try it again

timdengyun commented:
> > why I run ake generated, pkg/client/go.mod and pkg/client/go.sum also deleted? is there some change? modified: pkg/apis/nsx.vmware.com/v1alpha1/nsxserviceaccount_types.go modified: pkg/apis/nsx.vmware.com/v1alpha1/vpc_types.go modified: pkg/apis/nsx.vmware.com/v1alpha1/zz_generated.deepcopy.go modified: pkg/apis/nsx.vmware.com/v1alpha2/ippool_types.go deleted: pkg/client/go.mod deleted: pkg/client/go.sum
> 
> Yes, me too, fixed it, try it again

It's ok now. Approved.
